frontend/src/components/MenuBar.js

```diff
@@ -6,6 +6,9 @@ import Typography from '@material-ui/core/Typography';
 import IconButton from '@material-ui/core/IconButton';
 import HelpIcon from '@material-ui/icons/Help';
 
+/**
+ * Establishes style to use on rendering menu bar component
+ */
```

Comment:
I think one line javadocs should just be /** comment */

Reply:
Fixed in the MenuBar.js file and Form.js file, thank you!